### PR TITLE
Create Knack > Moped Migration Scripts (#8067)

### PR DESCRIPTION
**Pull request by johnclary**
WIP

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/8067

### CODE REVIEW

**Comment by johnclary** [comment]
@sergiogcx this is still very drafty, but I was able to make some progress formatting the knack data into moped mutations. For the most part this should be straightforward (thanks to graphql), but I will definitely need to spend some time with you to go over a few challenges and concerns. Most importantly, making sure we can roll back. 

I'll reach out to you about it when i'm back in the office next week.

File: moped-toolbox/knack_migration/project_template.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+"""Utils to create project mutation objects"""
+import uuid
+
+from requests.api import get
+
+
+def get_phase(phase_name, phase_start):
+    phases = {
+        "Unknown": 0,
```
Comment (johnclary) [inline moped-toolbox/knack_migration/project_template.py:9]: copied from local env. todo: double check these ID's match production

File: moped-toolbox/knack_migration/project_template.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        "completed": False,
+        "phase_start": phase_start,
+    }
+
+
+def get_component(signal):
+    uid = str(uuid.uuid4())
+    return {
+        "name": "Signal",
+        "description": "Signal - Traffic",
```
Comment (johnclary) [inline moped-toolbox/knack_migration/project_template.py:41]: Todo: set signal as type "Traffic" or "Pedestrian"

File: moped-toolbox/knack_migration/project_template.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        "component_id": 18,
+        "status_id": 1,
+        "moped_proj_features_components": {
+            "data": [
+                {
+                    "name": "Signal",
+                    "description": "Signal - Traffic",
```
Comment (johnclary) [inline moped-toolbox/knack_migration/project_template.py:48]: again, check signal type

**Comment by sergiogcx** [comment]
@johnclary sounds good! I'll start working on ways to identify and apply rollbacks reliably and accurately! This is looking good!

**Pull request closed without merging**